Dockerfile

```diff
@@ -6,7 +6,7 @@ RUN go mod download
 ADD . .
 RUN go build -o /src/pat
 
-FROM gcr.io/distroless/static
+FROM scratch
```

Comment:
Regarding this change.

> Statically compiled applications (Go) that do not require libc can use the gcr.io/distroless/static image, which contains:
> 
> ca-certificates
> A /etc/passwd entry for a root user
> A /tmp directory
> tzdata

So I'm going to look into if we need any of these.